lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -114,9 +114,16 @@ sub _get_ptero_shortcut_method {
 sub _get_ptero_execute_method {
     require Ptero::Builder::Job;
 
-
     my $self = shift;
     my $log_dir = shift;
+
+    my $guard;
+    if (my $lsb_sub_additional = Genome::Config::get('lsb_sub_additional')) {
+        my $old = $ENV{LSB_SUB_ADDITIONAL};
+        $guard = Scope::Guard->new(sub { $ENV{LSB_SUB_ADDITIONAL} = $old; });
+        $ENV{LSB_SUB_ADDITIONAL} = $lsb_sub_additional;
+    }
+
     my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
```

Comment:
`Genome::Config` variables must have values. (My original conditional was pointless, as the tests helpfully pointed out!)

Although the extra parameter might make sense for this particular variable, in general it's not clear to me when we would elect to use the supplied default in alternative to the existing value.